src/Parser/DeathKnight/Blood/Modules/Core/DeathsCaress.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import Analyzer from 'Parser/Core/Analyzer';
+import SPELLS from 'common/SPELLS/index';
+import SpellLink from 'common/SpellLink';
+import { formatPercentage } from 'common/format';
+
+import RuneTracker from '../../../Shared/RuneTracker';
+
+class DeathsCaress extends Analyzer {
+  static dependencies = {
+    tracker: RuneTracker,
+  };
+
+  dcCasts = 0;
+
+  on_byPlayer_cast(event) {
+    if (event.ability.guid !== SPELLS.DEATHS_CARESS.id) {
+      return;
+    }
+    this.dcCasts += 1;
+  }
+
+  get averageCastSuggestionThresholds() {
+    return {
+      actual: 1 - (this.dcCasts / (this.tracker.runesMaxCasts - this.tracker.runesWasted)),
```

Comment:
Threshold or the suggestion needs to be inverted (threshold is runes not spent on DC, suggestion is runes spent on DC). Otherwise good to merge.

Reply:
That's actually done on purpose.
I think having in the checklist "avoid casting DC" and display 100% with 0 casts is better than having "DC casts" and a full green bar with 0 casts.
Having the suggestion based on runes "wasted" by casting DC is better for the suggestions, at least in my opinion.

If you don't think so, tell me which one should he inverted and I'll update it.